### PR TITLE
Fixed sorting issue with Css Selectors with same specificity.

### PR DESCRIPTION
**Pull request by nsndeck**
In some cases when a lot css selectors are used unstable sort of CssSelectors results in problems with correct order of applying css selectors.

### CODE REVIEW

File: tns-core-modules/utils/utils-common.ts [hunk 1]
```diff
@@ -134,3 +134,36 @@ export function isDataURI(uri: string): boolean {
 
     return firstSegment && firstSegment.indexOf("data:") === 0 && firstSegment.indexOf('base64') >= 0;
 }
+
+export function mergeSort(arr, compareFunc) {
```
Comment (hdeshev) [inline tns-core-modules/utils/utils-common.ts]: Suggesting we rename this to `stableSort` to make it obvious why we need it :)

**Comment by vchimev** [comment]
run ci

**Comment by vchimev** [comment]
run ci

**Comment by nsndeck** [comment]
run ci

**Comment by dtopuzov** [comment]
run ci

**Pull request merged**